backend/views/core/invoices/view.py

```diff
@@ -48,10 +54,10 @@ def view(request, uuid):
         messages.error(request, "Invoice not found")
         return redirect("index")
 
-    try:
-        currency_symbol = request.user.user_profile.get_currency_symbol()
-    except UserSettings.DoesNotExist:
-        currency_symbol = "$"
+    if request.user != AnonymousUser():
+        currency_symbol = invoice.get_currency_symbol()
+    else:
+        currency_symbol = "£"
```

Comment:
Still using the wrong currency symbol for users that are not logged in. Either way should use invoice currency. Otherwise it's just the wrong currency as we're not doing a conversion.

Also instead of `request.user != AnonymousUser()` can't we just use `request.user.is_authenticated` to avoid confusion, cause `!= AnonymousUser()` in my opinion is harder to understand than is_authenticated

Reply:
hmm, let me see